Makefile

```diff
@@ -574,6 +574,10 @@ ifeq ($(API),true)
 	# Wait for Argo Server
 	until lsof -i :2746 > /dev/null ; do sleep 10s ; done
 endif
+ifeq ($(PROFILE),mysql)
+	# Wait for MySQL
+	until (: < /dev/tcp/localhost/3306) ; do sleep 10s ; done
```

Comment:
Note: I'm intentionally not using `lsof -i :3306` like the lines above because that isn't sufficient to prevent `connection refused` errors, because `lsof -i :3306` only tells you if something is listening on port 3306. I believe those errors happening in between the time `mysqld` binds to port 3306 and when it's ready to accept connections. Accessing `/dev/tcp/localhost/3306` actually opens a connection, and will fail if the connection is refused.